openfasoc/generators/cryo-gen/Makefile

```diff
@@ -1,39 +1,47 @@
 # ==============================================================================
```

Comment:
Can you add a help option to this makefile as we have for temp-sense-gen and ldo-gen

Reply:
Yes, help option is added.